doc/getting_started/index.rst

```diff
@@ -159,6 +159,14 @@ Set Up a Toolchain
    setting :envvar:`ZEPHYR_TOOLCHAIN_VARIANT` appropriately for a non-SDK
    toolchain.
 
+   As another way to skip setting up a toolchain, you may be able for
+   some configurations (e.g.: non-MCU x86 and x86_64 boards on Linux) to
+   re-use the development tools provided by your operating system that
+   `you already installed in the first step`__. To try this option use:
```

Comment:
This link is missing a target defined somewhere else in the doc.  I suspect you should change this to:
```suggestion
   you already installed in the first `host_setup`_ step. To try this option use:
```